sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -402,191 +210,23 @@ For more information and samples using prebuilt models see:
 ### Train a Model
```

Comment:
Build a Model ?

Reply:
I am not making modifications to the Readme in this PR. Will do in a future PR